pom.xml

```diff
@@ -371,6 +371,7 @@
 				<module>spring-cloud-gcp-kms</module>
 
 				<module>spring-cloud-previews</module>
+				<module>spring-cloud-generator</module>
```

Comment:
Is it going to be release automatically if we add it here?

Reply:
See https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686#discussion_r1169251614.